src/api/modules/utils.ts

```diff
@@ -129,81 +128,94 @@ export const appIdFromRegistry = (app: string, majorLocator: string) => {
-export function optionsFormatter(billingOptions: BillingOptions) {
+const FREE_BILLING_OPTIONS_TYPE = 'free'
+export const isFreeApp = (type?: string, free?: boolean) => type === FREE_BILLING_OPTIONS_TYPE || free
+
+type BillingInfo = {
+  subscription?: number
+  currency?: string
+  termsURL?: string
+  metrics?: Array<{
+    name: string
+    ranges: Range[]
+  }>
+}
+
+const chalkBillingTable = (table: any, { subscription, metrics, currency }: BillingInfo) => {
+  table.push([{ content: chalk.bold('Billable item') }, { content: chalk.bold('Pricing') }])
+
+  if (subscription) {
+    table.push([
+      {
+        content: `Subscription (monthly)`,
+      },
+      {
+        content: `${subscription} (${currency})`,
+      },
+    ])
+  }
+  metrics?.forEach(({ name, ranges }) => {
+    table.push([
+      {
+        content: name,
+      },
+      {
+        content: ranges.reduce<string>((text, { exclusiveFrom, multiplier }) => {
+          text += `${text.length > 0 ? `\n` : ''}${multiplier} (${currency}) per unit`
+          if (exclusiveFrom > 0) {
+            text += ` - for ${exclusiveFrom + 1} or more units`
+          }
+          return text
+        }, ''),
+        hAlign: 'left',
+      },
+    ])
+  })
+}
+
+const billingInfoFromPolicy = ({ currency, billing: { items } }: Policy): BillingInfo => {
+  const subscription = items.reduce<number>((sum, { fixed }) => (fixed ? sum + fixed : sum), 0)
+
+  const metrics = items.reduce((metricsInfo, { calculatedByMetricUnit }) => {
+    if (calculatedByMetricUnit) {
+      const { metricName, ranges } = calculatedByMetricUnit
+      metricsInfo.push({ name: metricName, ranges })
+    }
+    return metricsInfo
+  }, [])
+  return { currency, subscription, metrics }
+}
+
+const billingInfoFromPlan = ({ currency, price: { subscription, metrics } }: Plan): BillingInfo => ({
+  currency,
+  subscription,
+  metrics: metrics.map(({ id, ranges }) => ({ name: id, ranges })),
+})
+
+const buildBillingInfo = (plans?: Plan[], policies?: Policy[]): BillingInfo => {
+  if (plans && plans.length > 0) {
+    return billingInfoFromPlan(plans[0]) // Currenly only a single plan is supported in App Store
+  }
+  if (policies && policies.length > 0) {
+    return billingInfoFromPolicy(policies[0])
+  }
+}
+
+export function optionsFormatter({ plans, policies, type, free }: BillingOptions, licenseURL?: string) {
   /** TODO: Eliminate the need for this stray, single `cli-table2` dependency */
   const table = new Table({
     head: [{ content: chalk.cyan.bold('Billing Options'), colSpan: 2, hAlign: 'center' }],
     chars: { 'top-mid': '─', 'bottom-mid': '─', 'mid-mid': '─', middle: ' ' },
   })
-
-  if (billingOptions.free) {
-    table.push([{ content: chalk.green('This app is free'), colSpan: 2, hAlign: 'center' }])
+  const billingInfo = buildBillingInfo(plans, policies)
+  if (isFreeApp(type, free) || !billingInfo) {
+    table.push([{ content: chalk.green('Free app'), colSpan: 2, hAlign: 'center' }])
```

Comment:
How about using the colors from [Colors.ts](https://github.com/vtex/toolbelt/blob/60bb7edd56/src/api/constants/Colors.ts) ? 
Ex: ```chalk.hex(COLORS.GREEN)('Free app')```
And probably add new `ColorifyConstants` `readonly`  ? (I'm not sure about this, just a possibility)

Reply:
Done in 1a1f01e7e948ad1dcbe61c6a7cc761f9a28c6dd4, thanks!